test/unit/Site.test.js

```diff
@@ -351,6 +353,86 @@ test('Site read correct user defined variables', async () => {
   expect(subsub.number).toEqual('9999');
 });
 
+test('Site convert generates correct assets', async () => {
+  const json = {
+    'src/template/page.ejs': PAGE_EJS,
+
+    'inner/index.md': INDEX_MD_DEFAULT,
+
+    'inner/_Footer.md': '',
+    'inner/_Sidebar.md': '',
+    'inner/Home.md': '',
+  };
+  fs.vol.fromJSON(json, '');
+  await Site.initSite('inner/');
```

Comment:
Since we don't have a functional test for site conversion, could we check for `site.json` and `_markbind` directory here?

Reply:
I added in a functional test for site conversion, could you help me take a look 😅

Reply:
The functional test checks for the built converted site but not the converted files. Sorry if i wasn't clear.